### ISSUE TITLE
Project Setup Err.

### ISSUE THREAD

**Issue by cabbeer** [luochuanyuewu/nextus#1]
Hi, 
I followed the instructions for project setup but I keep running into the following issue, I believe it has to do with not 

I believe it has to do with the following step: 
Add a top navigation with the slug "main" and a footer navigation with the slug "footer"

```
- error components/navigation/Navbar.tsx (60:32) @ items
- error TypeError: Cannot read properties of undefined (reading 'items')
    at Navbar (./components/navigation/Navbar.tsx:82:55)
digest: "2484553377"
  58 |               className='menu dropdown-content rounded-box menu-sm z-50 mt-3 w-52 bg-base-100 p-2 shadow'
  59 |               mobile
> 60 |               items={navigation.items}
     |                                ^
  61 |             />
  62 |           </div>
  63 |           <Link href='/' className='btn btn-ghost text-xl normal-case'>
```

**Comment by co2nunes** [comment]
I have exactly the same issue.
In my case I can't create anything in the Language colletion, after the code and language item, it saves no record. I can't set it to the navigation items.

**Comment by luochuanyuewu** [comment]
Sorry for the delay, I will check it soon.

**Comment by luochuanyuewu** [comment]
> I have exactly the same issue. In my case I can't create anything in the Language colletion, after the code and language item, it saves no record. I can't set it to the navigation items.



> Hi, I followed the instructions for project setup but I keep running into the following issue, I believe it has to do with not
> 
> I believe it has to do with the following step: Add a top navigation with the slug "main" and a footer navigation with the slug "footer"
> 
> ```
> - error components/navigation/Navbar.tsx (60:32) @ items
> - error TypeError: Cannot read properties of undefined (reading 'items')
>     at Navbar (./components/navigation/Navbar.tsx:82:55)
> digest: "2484553377"
>   58 |               className='menu dropdown-content rounded-box menu-sm z-50 mt-3 w-52 bg-base-100 p-2 shadow'
>   59 |               mobile
> > 60 |               items={navigation.items}
>      |                                ^
>   61 |             />
>   62 |           </div>
>   63 |           <Link href='/' className='btn btn-ghost text-xl normal-case'>
> ```
Fixed in #146486626952d55282247066e910a9654e4abd2c

**Comment by luochuanyuewu** [comment]
> I have exactly the same issue. In my case I can't create anything in the Language colletion, after the code and language item, it saves no record. I can't set it to the navigation items.

Turns out, it was the environment variable 'CACHE_ENABLED=true' in Directus that caused this issue. Currently, you should change 'CACHE_ENABLED' to 'false'
